test/loaders/handlebars.js

```diff
@@ -0,0 +1,37 @@
+/*
+ * This file is part of the Symfony Webpack Encore package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+'use strict';
+
+const expect = require('chai').expect;
+const WebpackConfig = require('../../lib/WebpackConfig');
+const RuntimeConfig = require('../../lib/config/RuntimeConfig');
+const handlebarsLoader = require('../../lib/loaders/handlebars');
+
+function createConfig() {
+    const runtimeConfig = new RuntimeConfig();
+    runtimeConfig.context = __dirname;
+    runtimeConfig.babelRcFileExists = false;
+
+    return new WebpackConfig(runtimeConfig);
+}
+
+describe('loaders/handlebars', () => {
+    it('getLoaders() basic usage', () => {
+        const config = createConfig();
+        config.enableHandlebarsLoader(function(options) {
+            options.debug = true;
+        });
+
+        const actualLoaders = handlebarsLoader.getLoaders(config);
+        expect(actualLoaders).to.have.lengthOf(1);
+        // callback is used
+        expect(actualLoaders[0].options.debug).to.be.true;
+    });
```

Comment:
Could you also add a test there to check if returning a value from the callback overrides the previous options correctly?

For instance:

```js
config.enableHandlebarsLoader(function(options) {
  options.debug = true;

  // This should override the original config
  return { foo: true };
});

// (...)

expect(actualLoaders[0].options.debug).to.be.undefined;
expect(actualLoaders[0].options.foo).to.be.true;
```